mappings/net/minecraft/world/gen/feature/util/DensityFunctions.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_6954 net/minecraft/world/gen/feature/util/DensityFunctions
+	METHOD method_41100 initAndGetDefault ()Lnet/minecraft/class_6880;
```

Comment:
This name is inconsistent with similar methods passed as the default getter supplier in other registries.